docs/content/connectors/mongodb-cdc.md

```diff
@@ -0,0 +1,440 @@
+    </tr>
+    <tr>
+      <td>copy.existing.pipeline</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td> An array of JSON objects describing the pipeline operations to run when copying existing data.<br>
+           This can improve the use of indexes by the copying manager and make copying more efficient.
+           eg. <code>[{"$match": {"closed": "false"}}]</code> ensures that 
+           only documents in which the closed field is set to false are copied.
+      </td>
+    </tr>
+    <tr>
+      <td>copy.existing.max.threads</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">Processors Count</td>
+      <td>Integer</td>
+      <td>The number of threads to use when performing the data copy.</td>
+    </tr>
+    <tr>
+      <td>copy.existing.queue.size</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">16000</td>
+      <td>Integer</td>
+      <td>The max size of the queue to use when copying data.</td>
+    </tr>
+    <tr>
+      <td>poll.max.batch.size</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">1000</td>
+      <td>Integer</td>
+      <td>Maximum number of change stream documents to include in a single batch when polling for new data.</td>
+    </tr>
+    <tr>
+      <td>poll.await.time.ms</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">1500</td>
+      <td>Integer</td>
+      <td>The amount of time to wait before checking for new results on the change stream.</td>
+    </tr>
+    <tr>
+      <td>heartbeat.interval.ms</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">0</td>
+      <td>Integer</td>
+      <td>The length of time in milliseconds between sending heartbeat messages. Use 0 to disable.</td>
+    </tr>
+    </tbody>
+</table>
+</div>
+
+Note: `heartbeat.interval.ms` is highly recommended to set a proper value larger than 0 if the collection changes slowly.
+The heartbeat event can pushing the `resumeToken` forward to avoid `resumeToken` being expired when we recover the Flink job from checkpoint or savepoint.
+
+Features
+--------
+
+### Exactly-Once Processing
+
+The MongoDB CDC connector is a Flink Source connector which will read database snapshot first and then continues to read change stream events with **exactly-once processing** even failures happen. 
+
+### Snapshot When Startup Or Not
+
+The config option `copy.existing` specifies whether do snapshot when MongoDB CDC consumer startup. Defaults to `true`.
+
+### Snapshot Data Filters
+
+The config option `copy.existing.pipeline` describing the filters when copying existing data.
+This can improve the use of indexes by the copying manager and make copying more efficient.
+
+In the following example, the $match aggregation operator ensures that only documents in which the closed field is set to false are copied.
+`copy.existing.pipeline=[ { "$match": { "closed": "false" } } ]`
```

Comment:
Maybe we should quotes `$match`, this part of the rendering is broken.

![image](https://user-images.githubusercontent.com/5378924/132851942-3393105a-ec79-4924-900a-02a44f35000a.png)

Reply:
Fixed.